src/buttons/button.scss

```diff
@@ -160,7 +160,7 @@
   &:hover,
   [open] > & {
     color: var(--color-text-inverse);
-    background-color: var(--color-btn-#{$type}-bg-hover);
+    background-color: var(--color-btn-#{$type}-bg-hover); // stylelint-disable-line primer/no-undefined-vars
```

Comment:
I don't love having to ignore these variables but there's no way we can lint them until they're built... a problem for another PR